.gitlab-ci.yml

```diff
@@ -178,17 +178,19 @@ test:staging-deployment:
 publish:tests:
   stage: publish
   except:
-    - /^saas-[a-zA-Z0-9.]+$/
+    - /^(staging|saas-[a-zA-Z0-9.]+)$/
```

Comment:
Why do you want to except `staging`. It is reported in other repos so that a PR to staging can still measure coverage.

Reply:
the tests should never change between master and staging, so the coverage will be the same for _master@commit_ - I plan to exempt the unit tests job from staging in a different PR...

Reply:
> the tests should never change between master and staging

That is quite an statement. For consistency, we should first add `except:staging` branch from this job in the template, then follow-up down to the repos that have their own coverage publishing (like gui). And of course remove the possible PR checks on staging on all the repos.

In any case, quite unrelated to this PR so go ahead with the merge.